Games/types/Mafia/roles/cards/WinIfDiesWithDoll.js

```diff
@@ -0,0 +1,29 @@
+const Card = require("../../Card");
+const { PRIORITY_WIN_CHECK_DEFAULT } = require("../../const/Priority");
+
+module.exports = class WinIfDiesWithDoll extends Card {
+
+    constructor(role) {
+        super(role);
+        
+        this.winCheck = {
+            priority: PRIORITY_WIN_CHECK_DEFAULT,
+            againOnFinished: true,
+            check: function (counts, winners, aliveCount) {
+                if (this.data.dollDeath) {
+                    winners.addPlayer(this.player, this.name);
```

Comment:
not sure if it matters, creepy girl appears twice in the winner group

i don't really know why other roles check if group exists first